src/jit/gentree.h

```diff
@@ -3497,9 +3503,26 @@ struct GenTreeCall final : public GenTree
         return (gtCallMoreFlags & GTF_CALL_M_DOES_NOT_RETURN) != 0;
     }
 
+    bool IsFatPointerCandidate() const
+    {
+        return (gtCallMoreFlags & GTF_CALL_M_FAT_POINTER_CHECK) != 0;
+    }
+
     bool IsPure(Compiler* compiler) const;
 
-    unsigned short gtCallMoreFlags; // in addition to gtFlags
+    void SetFatPointerCandidate(bool value)
+    {
```

Comment:
I didn't find existing setter, maybe I was looking bad. Do we prefer set and clear?